src/de.rs

```diff
@@ -443,6 +448,64 @@ pub struct FutureIncompatReportConfig {
     pub frequency: Option<Value<Frequency>>,
 }
 
+/// The `[cargo-new]` table.
+///
+/// [reference](https://doc.rust-lang.org/nightly/cargo/reference/config.html#cargo-new)
+#[derive(Debug, Clone, Default, Serialize, Deserialize)]
+#[serde(rename_all = "kebab-case")]
+#[non_exhaustive]
+pub struct CargoNewConfig {
+    /// Specifies the source control system to use for initializing a new repository.
+    /// Valid values are git, hg (for Mercurial), pijul, fossil or none to disable this behavior.
+    /// Defaults to git, or none if already inside a VCS repository. Can be overridden with the --vcs CLI option.
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub vcs: Option<Value<VersionControlSoftware>>,
+}
+
+#[allow(clippy::exhaustive_enums)]
```

Comment:
I think this enum should use `#[non_exhaustive]`.

Reply:
Thanks, good catch. I made this `#[non_exhaustive]` and reexported it at the crate root to match the other enums :+1: